packages/screenreaderonly/src/react/index.js

```diff
@@ -0,0 +1,27 @@
+import { css } from 'glamor'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import stylesheet from '../css/index.js'
+
+const styles = {
+  screenReaderOnly: () => css(stylesheet['.psds-screen-reader-only'])
+}
+
+const ScreenReaderOnly = React.forwardRef((props, forwardedRef) => {
+  const { as: Tag, ...rest } = props
+  const ref = React.useRef()
+  React.useImperativeHandle(forwardedRef, () => ref.current)
```

Comment:
Why are we including this if we are not using the ref internally?

Reply:
because i copy/pasted. i'll fix